packages/integration-karma/test/rendering/slot-dont-recycle-vm/index.spec.js

```diff
@@ -0,0 +1,30 @@
+import { createElement } from 'lwc';
+import Container from 'x/container';
+
+describe('Component', () => {
+    it('should not have have an error accessing state.foo', function() {
```

Comment:
Do we expect an exception to occur if this fix regresses? If so, the test should indicate that using `.toThrow()`.  Also, what is `state.foo` referring to?